ci-operator/step-registry/storage/conf/csi-optional/aws-efs/storage-conf-csi-optional-aws-efs-chain.yaml

```diff
@@ -0,0 +1,27 @@
+chain:
+  as: storage-conf-csi-optional-aws-efs
```

Comment:
Can you please modify  [aws-efs-csi-driver-operator](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/aws-efs-csi-driver-operator/openshift-aws-efs-csi-driver-operator-master.yaml) + [openshift-e2e-aws-csi-efs-workflow](https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/e2e/aws/csi/efs/openshift-e2e-aws-csi-efs-workflow.yaml) to use this chain, so all EFS installation is on one place?

Reply:
Hmm, subscription would need to be different - in [aws-efs-csi-driver-operator](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/aws-efs-csi-driver-operator/openshift-aws-efs-csi-driver-operator-master.yaml) repo we want to use to just built operator index.

Reply:
@Phaow promised to fix it as a separate PR :-)

Reply:
Good point! Thanks! I'll reuse the new chain in our current test jobs to unify our efs install workflow in a new separate PR.